jwst/ramp_fitting/tests/test_ramp_fit.py

```diff
@@ -744,10 +744,11 @@ def test_twenty_groups_two_segments():
     np.testing.assert_allclose(oslope[wh_data], 10. / deltatime, rtol=1E-4)
     np.testing.assert_allclose(oyint[0, 0, 0, :], model1.data[0, 0, 0, :], rtol=1E-5)
 
-    np.testing.assert_allclose(
-        opedestal[0, 0, :],
-        model1.data[0, 0, 0, :] - 10.,
-        rtol=1E-5)
+    opedal = opedestal[0, 0, :]
```

Comment:
code style check claims that `opedal` is set but never used

Reply:
Fixed.